### DISCUSSION TITLE
Pimcore Copilot Select type Environment Variable Configuration

### DISCUSSION THREAD

**Discussion by Chauhan-Mukesh** [pimcore/pimcore#17088] [General]
Hello Pimcore community,

I am currently working on integrating **Pimcore Copilot's variant generator** into my project and have encountered a question regarding configuration flexibility. Specifically, I'd like to leverage the functionality of **Options Providers and Options Provider Data** within the **"Environment Variable"** configuration for a **Select Type Field**.

While I understand how to make the field mandatory or non-mandatory and provide static options, I am seeking guidance on how to dynamically populate the options using Options Providers. Is this currently supported within the Pimcore Copilot configuration framework?

If so, I would greatly appreciate it if someone could provide a sample configuration demonstrating how to achieve this. Any insights or code examples would be invaluable in streamlining my implementation.

Thank you in advance for your expertise and assistance.

**Comment by fashxp** [comment]
Hi, 
this is not supported yet, but actually a great idea. We should put that on our roadmap...

Cheers.
